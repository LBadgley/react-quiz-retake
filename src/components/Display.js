import React from 'react';
import PropTypes from 'prop-types';

export default function Display({ headerText, image, footerText }) {
  const styles = {
    width: '300px',
    height: '300px',
    margin: '20px',
    headerText,
    image,
    footerText,
  };

  return (
    <section style={styles}>
      <p >{headerText}</p>
      <div>
        <img src="http://preetkamal.com/wp-content/uploads/2019/03/Friday-Meme-Friday-Is-My-Second.jpg" alt="friday meme"></img>
      </div>
      <p>{footerText}</p>
    </section>
  );
}

Display.propTypes = {
  headerText: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired,
  footerText: PropTypes.string.isRequired
};

