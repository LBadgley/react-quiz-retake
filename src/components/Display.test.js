import React from 'react';
import { shallow } from 'enzyme';
import Display from './Display';

describe('Display Component text', () => {
  it('renders the display', () => {
    const wrapper = shallow(<Display />);
    expect(wrapper).toMatchSnapshot();
  });
});
