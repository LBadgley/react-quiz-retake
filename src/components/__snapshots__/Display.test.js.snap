// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Component text renders the display 1`] = `
<section
  style={
    Object {
      "footerText": undefined,
      "headerText": undefined,
      "height": "300px",
      "image": undefined,
      "margin": "20px",
      "width": "300px",
    }
  }
>
  <p />
  <div>
    <img
      alt="friday meme"
      src="http://preetkamal.com/wp-content/uploads/2019/03/Friday-Meme-Friday-Is-My-Second.jpg"
    />
  </div>
  <p />
</section>
`;
